ansible/roles/hypothesis/tasks/main.yml

```diff
@@ -139,5 +139,3 @@
     - user-created
     - user-passwd
     - user-deleted
-  tags:
```

Comment:
So this one is actually fun. That script under https://github.com/liquidinvestigations/setup/blob/fix/ansible/roles/hypothesis/files/hooks/user#L7 still has some jinja template in it. It's only used for the email generation, so all the hypothesis users we generate are `username@{{ liquid_domain }}`.

Either move them back to templates and keep this tag, or go to that file and make all emails end in `@liquid.investigations.bogus.email.org` or something

Reply:
Good catch, thanks.